phpunit.xml.old.dist

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit colors="true" bootstrap="tests/bootstrap.php">
+	<testsuites>
+        <testsuite name="SimplePie Test Suite">
+            <directory>./tests</directory>
+        </testsuite>
+        <testsuite name="Old Tests">
+			<file>tests/oldtests.php</file>
+        </testsuite>
+    </testsuites>
+
+	<filter>
+		<blacklist>
+			<directory suffix=".php">tests</directory>
+		</blacklist>
+		<whitelist>
+			<directory suffix=".php">SimplePie</directory>
+		</whitelist>
+	</filter>
+</phpunit>
```

Comment:
I need more proof that these files need to be different before accepting this.

Reply:
This file I will be removed and use the ```phpunit.xml.dist```.